mcs/class/corlib/ReferenceSources/CalendarData.cs

```diff
@@ -34,6 +34,36 @@ namespace System.Globalization
 	[StructLayout (LayoutKind.Sequential)]
 	partial class CalendarData
 	{
+		private static string[] HEBREW_MONTH_NAMES = new[] {
```

Comment:
Maybe a comment as to roughly what these are? Perhaps the identifier suffice.
or a comment saying ideally it is in native, to avoid fix startup.

Reply:
It's a hack anyway, we need to find out why our "ICU" analogue doesn't handle it and fix there (this PR just fixes CI)

Reply:
It doesn't handle it because the `calendarId` is completely ignored, both in managed code and the native code. It was there as TODO for couple of years - `TODO: Convert calendar-id to mono runtime calendar-id when it's used`. Basically any combination of locale and non-default calendar for given locale is broken.

No one noticed it until now because Hebrew is the only supported calendar with leap months and it was the first leap year since the code was introduced. The produced `DateTime.ToString` results for the given combination were incorrect since the very beginning but they were round-trippable because the same wrong names were used for parsing.